README.md

```diff
@@ -1,3 +1,29 @@
 ##### Credits
 
-Icons made by [Freepik](https://www.freepik.com) from [Flaticon](https://www.flaticon.com/).
\ No newline at end of file
+Icons made by [Freepik](https://www.freepik.com) from [Flaticon](https://www.flaticon.com/).
+
+
+### Contributing
+
+To build this project, first import it into visual studio.  You'll need to make a new file called Directory.Build.targets to specify file paths.
+
+In addition, if properly configured, this file will automatically drop the output of the build into your derail valley mods folder.  UMM is still required to exist, but the info.json and dll will be enough to continue to develop.
+
+An example of what the targets file should look like is provided:
```

Comment:
If you would, please credit this to yourself or wherever the original source is. I don't want to have to support this or field questions from people having difficulty with it, since I develop entirely in WSL and VSCode.

Reply:
This has simply been removed - future contributors can get help on discord